backend/api/axios-config-decorators.test.ts

```diff
@@ -1,11 +1,11 @@
-const decorators = require('./axios-config-decorators')
-const contextProperties = require('../contextProperties')
+import * as contextProperties from '../contextProperties'
+import * as decorators from './axios-config-decorators'
```

Comment:
might be beyond the scope of what you're working on but would be nice if you could get rid of axios and solely use superagent.. not sure exactly where you're using it the template project uses this [client](https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/data/hmppsAuthClient.ts) for talking to auth for token purposes.